frontend/src/app/order/order.routes.ts

```diff
@@ -16,7 +16,7 @@ export const ORDER_ROUTES: Routes = [
       {
         path: '',
         pathMatch: 'full',
-        canActivate: [OrdersGuard],
+        canActivate: [ordersGuard],
```

Comment:
this could be refactored as a 'CanMatch' no?

Reply:
Maybe. I tried once to use CanMatch and it ended up in an infinite loop, I have no idea why. Since this works fine, I'll leave it this way.